docs/detections/detection-engine-intro.asciidoc

```diff
@@ -88,6 +88,12 @@ Detection rules::
 Background tasks that run periodically and produce alerts when suspicious
 activity is detected.
 
+[[term-sec-indices]]
+{es-sec} indices::
+Indices containing host and network source events (such as
+`packetbeat-*`, `log-*`, and `winlogbeat-*`). Detection rules use these
```

Comment:
Nit: detection rules _can_ use these indices, but it is not required. A new rule defaults to using those defined in the `securitySolution:defaultIndex` UI setting, but it can be overridden at both levels. We may want to mention that setting and this behavior here.